src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -1025,10 +990,90 @@ private void CheckNewModifier(Symbol symbol, bool isNew, DiagnosticBag diagnosti
                 suppressAccessors = true;
             }
 
+            return;
+
             bool isOrContainsErrorType(TypeSymbol typeSymbol)
             {
                 return (object)typeSymbol.VisitType((currentTypeSymbol, unused1, unused2) => currentTypeSymbol.IsErrorType(), (object)null) != null;
             }
+
+            static void checkValidNullableMethodOverride(
+                Location overridingMemberLocation,
+                MethodSymbol overriddenMethod,
+                MethodSymbol overridingMethod,
+                DiagnosticBag diagnostics)
+            {
+                var compilation = overridingMethod.DeclaringCompilation;
+                if (overriddenMethod is null ||
+                    overridingMethod is null ||
+                    compilation is null ||
+                    overridingMethod.IsImplicitlyDeclared ||
```

Comment:
>overridingMethod.IsImplicitlyDeclared [](start = 20, length = 37)

What about accessors for an expession bodied auto property? It looks like the original code was suppressing the check only for IsImplicitlyDeclared regular methods vs. accessors. I would duplicate the logic, or remove this condition completely.